stress/common/src/main/java/alluxio/stress/BaseParameters.java

```diff
@@ -86,4 +87,8 @@
 
   @Parameter(names = {"-h", HELP_FLAG}, help = true)
   public boolean mHelp = false;
+
+  @Parameter(names = {START_DELAY_MS},
```

Comment:
You can make it a string value here like
https://github.com/Alluxio/alluxio/blob/58555b7de4633c6ac57e20ae73ded7c169a6f8f9/stress/common/src/main/java/alluxio/stress/rpc/RpcBenchParameters.java#L26